src/platforms/javascript/common/install/loader.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: Loader Script
+sidebar_order: 10
+redirect_from:
+  - /platforms/javascript/loader/
+  - /platforms/javascript/install/lazy-load-sentry/
+  - /platforms/javascript/install/cdn/
+description: "Learn about the Sentry JavaScript Loader Script"
+---
+
+The Loader is a small script that takes care for you initializing the SDK for you, as well as keep it up-to-date. It is desgined as a drop in solution that you don't have to think about a lot.
+
+## Using the Loader
+
+
+To use the loader, go in the Sentry UI to _Settings -> Projects -> Client Keys (DSN)_, and then press the _Configure_ button. Copy the script tag from the _JavaScript Loader_ section, and include it as the first script on your page. (By including it first, you allow it to catch and buffer events from any subsequent scripts, while still ensuring the full SDK doesn't load until after everything else has run.)
```

Comment:
```suggestion
To use the loader, go in the Sentry UI to **Settings > Projects > (select project) > Client Keys (DSN)**, and then press the "Configure" button. Copy the script tag from the "JavaScript Loader" section and include it as the first script on your page. By including it first, you allow it to catch and buffer events from any subsequent scripts, while still ensuring the full SDK doesn't load until after everything else has run.
```

I realized after making this tweak that this is just copied from the original page, but it would still be nice to tweak it to fit our style guide :D